README.md

```diff
@@ -37,7 +37,7 @@ Next-generation ES module bundler for <a href="https://deno.land/">Deno</a> port
 
 [Rollup](https://github.com/rollup/rollup) is a module bundler for JavaScript which compiles small pieces of code into something larger and more complex, such as a library or application.
 
-This library extends Rollup so that it can be used within Deno scripts to bundle modules.
+This library extends Rollup so that it can be used in Deno. Rollup is designed for use in NodeJS, with all the code on the local file-system, and dependent packages pre-installed in `node_modules`. Deno knows nothing about packages or `node_modules`, but it can load modules from both the local file-system and remote URLs. So deno-rollup uses the browser version of Rollup, using `file://` URLs to access the local file-system. Many CJS modules can be loaded via a proxy server such as JSPM or `esm.sh`, which convert CJS to ES modules. Package references can be mapped to URLs using the import map plugin (see below). The end result is that external dependencies can be included in a build without having to be installed on the local file-system.
```

Comment:
So... I wonder if some of this detail would be better placed in a different documentation like section regarding differences to the Node version etc. IMO the overview is a brief tagline about what this library does, whereas we're going into some detail here about how Deno differs from NodeJS ( which is kind of separate to this particular lib ) and some of the internals of deno-rollup 🙃 

I started to add some extra detail to the `Documentation` section - perhaps we could start fleshing that out with some subsections?

> using `file://` URLs to access the local file-system.

It's worth noting that `deno-rollup` doesn't just use `file://` URLs to access the locla file-system, it supports relative and absolute path strings as well.

> Many CJS modules can be loaded via a proxy server such as JSPM or `esm.sh`, which convert CJS to ES modules

This is just general guidance around how to use modules in Deno rather than anything specific to deno-rollup. Not against having it in here, but again feels like something for a different section at least - it is something that users should be able to get from reading the Deno docs ( see https://deno.land/#third-party-modules ). If we're going to list CDN providers, should probably also include Skypack in there as well ( though ultimately users can and should be encouraged to use whatever they like, don't want to endorse certain providers, or imply certain ones must be used for deno-rollup to work ).

>  Package references can be mapped to URLs using the import map plugin (see below).

There also X other things that can be done with deno-rollup plugins, so feels a little odd perhaps to call out this specfic plugin without also mentioning how we can support `json` and `css` imports etc. ( even though json modules and import assertions haven't landed yet ).

> The end result is that external dependencies can be included in a build without having to be installed on the local file-system

This is potentially misleading in the sense that Deno will locally cache all remote modules afaik ( may be wrong? )

Reply:
all of what you say is true, but I think the basic question is 'why use deno-rollup?' Bundling/minifying is one of the most widely used JS utilities, and Rollup and other Node-based packages are well established. So why bother with deno-rollup? Much of the answer is going to be related to 'why use Deno rather than Node' rather than 'why use deno-rollup and not other deno bundlers'. And I think it's best to address both questions by emphasising what deno-rollup can do rather than what other libraries/packages can't.

I'd be happy with there being a separate docs section on how to best use this library - perhaps a tutorial?

Reply:
```suggestion
This library extends Rollup so that it can be used in Deno - supporting core Deno features out-of-the-box such as URL imports, Typescript and JSX.
```

Going to simplify the addition down to mentioning the core "Deno" related differences to the o.g. Rollup for now - one to have a think on. A tutorial would be really nice if that is something you'd be game for 😄